### PR TITLE
Add contribution guidelines and code of conduct

### PR DESCRIPTION
**Pull request by urschrei**
See [here](https://github.com/orgs/georust/teams/core/discussions/2) for prior discussion. This is a first effort, adapted from the originals at https://github.com/rust-lang/rust. I haven't specified any humans as part of the mod team, as this doesn't really make sense for us  – I've just said "contact the  the core team". Maybe we'd prefer specifically contactable people?

### CODE REVIEW

File: geo/src/algorithm/CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# The Rust-Geo Code of Conduct
```
Comment (frewsxcv) [inline geo/src/algorithm/CODE_OF_CONDUCT.md]: Is our CoC going to cover all GeoRust projects? If so, should this be the GeoRust CoC?

File: geo/src/algorithm/CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+And if someone takes issue with something you said or did, resist the urge to be defensive. Just stop doing what it was they complained about and apologize. Even if you feel you were misinterpreted or unfairly accused, chances are good there was something you could've communicated better — remember that it's your responsibility to make your fellow Rustaceans comfortable. Everyone wants to get along and we are all here first and foremost because we want to talk about cool technology. You will find that people will be eager to assume good intent and forgive as long as you earn their trust.
+
+*Adapted from the [Node.js Policy on Trolling](http://blog.izs.me/post/30036893703/policy-on-trolling) as well as the [Contributor Covenant v1.3.0](https://www.contributor-covenant.org/version/1/3/0/).*
+
+[mod_team]: https://github.com/orgs/georust/teams/core
```
Comment (frewsxcv) [inline geo/src/algorithm/CODE_OF_CONDUCT.md]: I'm not entirely sure about this, so I can check later, but I don't think non-organization-members can access this page. I was also considering registering georust.org and setting up an alias like code-of-conduct@georust.org that would get sent to some of us, but haven't had the time.

File: geo/src/algorithm/CODE_OF_CONDUCT.md (hunk 1 above)
Comment (urschrei) [reply geo/src/algorithm/CODE_OF_CONDUCT.md]: I think that's a good idea – should we ask individual project owners?

File: geo/src/algorithm/CODE_OF_CONDUCT.md (hunk 2 above)
Comment (urschrei) [reply geo/src/algorithm/CODE_OF_CONDUCT.md]: I just checked, and you're right – you can't see teams. A `.org` and dedicated email is a good idea for this. I'm happy to do it during the week and send over the creds if you're too busy.

**Comment by frewsxcv** [comment]
I registered georust.org and setup mods@georust.org to forward to a private Google Group

**Comment by urschrei** [comment]
Amazing! Shall we merge?

**Pull request merged**